server/settings/creators.py

```diff
@@ -0,0 +1,75 @@
+from ayon_server.settings import (
+    BaseSettingsModel,
+    SettingsField,
+)
+
+
+class BasicCreatorModel(BaseSettingsModel):
+    enabled: bool = SettingsField(title="Enabled")
+    default_variants: list[str] = SettingsField(
+        default_factory=list,
+        title="Default Variants"
+    )
+
+
+class CreatorsModel(BaseSettingsModel):
+
+    CreateAction: BasicCreatorModel = SettingsField(
+        default_factory=BasicCreatorModel,
+        title="Create Action"
+    )
+    CreateAnimation: BasicCreatorModel = SettingsField(
+        default_factory=BasicCreatorModel,
+        title="Create Animation"
+    )
+    CreateBlendScene: BasicCreatorModel = SettingsField(
+        default_factory=BasicCreatorModel,
+        title="Create BlendScene"
+    )
+    CreateCamera: BasicCreatorModel = SettingsField(
+        default_factory=BasicCreatorModel,
+        title="Create Camera"
+    )
+    CreateLayout: BasicCreatorModel = SettingsField(
+        default_factory=BasicCreatorModel,
+        title="Create Layout"
+    )
+    CreateModel: BasicCreatorModel = SettingsField(
+        default_factory=BasicCreatorModel,
+        title="Create Model"
+    )
+    CreatePointCache: BasicCreatorModel = SettingsField(
```

Comment:
```suggestion
    CreatePointcache: BasicCreatorModel = SettingsField(
```

This should be lowercase `c` for the settings to actually work as @LiborBatek pointed out it currently does not work.